### PR TITLE
explaining how user can specify mysql root user\'s password in non-default cases

### PR DESCRIPTION
**Pull request by dharley-5am**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,32 +197,32 @@ Testing
 ###Run the Validation Suite as part of install
 At the end of the mvn install process, an embedded GlassFish instance will start and the Validation Suite will run against it:
         $ cd <CONNECT_CLONE_DIR>/Product/SoapUI_Test/ValidationSuite
-        $ mvn clean install
+        $ mvn clean install -Dmysql.root.password=<password>
+
+###Run the Validation Suite via Maven script
+The Validation Suite can be run via a Maven script against a standalone installation of the application server:
+        $ cd <CONNECT_CLONE_DIR>/Product/SoapUI_Test/ValidationSuite
+        $ mvn verify -Dstandalone -Dproperties.dir=<applicaiton server configuration dir> -Dmysql.root.password=<password>
```
Comment (tparnell) [inline README.md]: Why not just put all the properties into the list below (starting at line 207?)

File: README.md [hunk 2]
```diff
@@ -58,15 +65,20 @@ OSX / Linux Users:
 Everyone:
 
         $ cd <CONNECT_CLONE_DIR>
+
+        If using default CONNECT MySQL root user password "NHIE-Gateway":
```
Comment (tparnell) [inline README.md]: This line repeats info from 48-54.  I don't think it's worth repeating every time there is a maven command line to tell the user about this option.

File: README.md [hunk 3]
```diff
@@ -58,15 +65,20 @@ OSX / Linux Users:
...
         $ mvn clean install
 
+        If using non-default CONNECT MySQL root user password:
```
Comment (tparnell) [inline README.md]: Remove.

File: README.md [hunk 4]
```diff
@@ -79,11 +91,16 @@ and choose the clone repo directory (\<CONNECT_CLONE_DIR\>), e.g. CONNECT. You m
 * \<CONNECT_CLONE_DIR\>/Product/Production
 
 ####Building an ear
-All services profiles are active by default, so to build an ear containging all services, just execute:
+All services profiles are active by default, so to build an ear containing all services, using the default MySQL root password, just execute:
 
         $ cd <CONNECT_CLONE_DIR>
         $ mvn clean install
 
+To build an ear containing all services, with non-default MySQL root user password, just execute:
```
Comment (tparnell) [inline README.md]: Similar to above, I don't think all this repetition is necessary.

File: README.md (hunk 1 above)
Comment (dharley-5am) [inline README.md]: good idea, thanks.

**Pull request merged**